op/pulse.c

```diff
@@ -33,6 +35,8 @@ static pa_channel_map		 pa_cmap;
 static pa_cvolume		 pa_vol;
 static pa_sample_spec		 pa_ss;
 
+static bool 		 is_pipewire = false;
+
```

Comment:
The indentation is off since our tabs are 8-wide. There's a similar mismatch below in the alignment of `_pa_server_info_cb()`'s params.